commit cce07e6d21fa870d7a307108fb67644b56a86229 (refs/changes/28/5228/1)
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2018-12-11T10:24:11-07:00 (10 months ago)
    
    joyent/node-fast#22 Fix copy-paste bug in fast_server module

diff --git a/lib/fast_server.js b/lib/fast_server.js
index 1c633cb..4cc50c0 100644
--- a/lib/fast_server.js
+++ b/lib/fast_server.js
@@ -227,7 +227,7 @@ function FastServer(args)
 	this.fs_nrequests_failed = 0;		/* count of reqs failed */
 
 	if (this.fs_collector) {
-		if (this.fc_collector.FIXED_BUCKETS === true) {
+		if (this.fs_collector.FIXED_BUCKETS === true) {
 			fixed_buckets = true;
 		}
 		this.fs_request_counter = this.fs_collector.counter({
